.github/.copilot/breadcrumbs/2025-01-15-1400-crp-delete-policy.md

```diff
@@ -0,0 +1,79 @@
+# CRP Delete Policy Implementation
+
+## Problem Analysis
+
+The issue requests adding API options to allow customers to choose whether to delete placed resources when a CRP (ClusterResourcePlacement) is deleted.
+
+### Current Deletion Behavior
+1. When a CRP is deleted, it has two finalizers:
+   - `ClusterResourcePlacementCleanupFinalizer` - handled by CRP controller to delete snapshots
+   - `SchedulerCleanupFinalizer` - handled by scheduler to delete bindings
+   
+2. The deletion flow:
+   - CRP controller removes snapshots (ClusterSchedulingPolicySnapshot, ClusterResourceSnapshot)
+   - Scheduler removes bindings (ClusterResourceBinding) which triggers cleanup of placed resources
+   - Currently there's no option for users to control whether placed resources are deleted
+
+### References
+- Kubernetes DeleteOptions has `PropagationPolicy` with values: `Orphan`, `Background`, `Foreground`
+- AKS API has `DeletePolicy` pattern (couldn't fetch exact details but following similar pattern)
+
+## Implementation Plan
+
+### Phase 1: API Design
+- [x] Add `DeletionPolicy` field to `PlacementSpec` in beta API
+- [x] Define enum values: `Delete` (default), `Orphan`
+- [x] Update API documentation and validation
+
+### Phase 2: Controller Logic Updates
```

Comment:
The whole part seems to be wrong. Could you keep "Phase 2", remove its content, add a TODO for @Arvindthiru to fill out the details.